google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -71,13 +71,22 @@ public long getActualOffset() {
     }
   }
 
-  /** Stream has already been finalized. */
+  /**
+   * Stream created by CreateWriteStream has already been finalized. You need to recreate a stream
```

Comment:
The write stream has already been finalized and will not accept further appends.   To send additional
appends, you will need to create a new write stream via CreateWriteStream.